path_utility.cpp

```diff
@@ -130,7 +130,7 @@ void initialize_filesystem()
 /// Notably, path("/bin/sh/") succeeds, silently discarding the
 /// trailing '/'.
 
-fs::path LMI_SO modify_directory
+fs::path modify_directory
```

Comment:
Why has `LMI_SO` been removed here? AFAICS this function is used from the test and this shouldn't link if it's not visible.

Reply:
I removed `LMI_SO` from .cpp files because in `.h` files `LMI_SO` already exists. And most of functions declared with `LMI_SO` in headers and don't uses `LMI_SO` in implementations in source fies.

Reply:
Sorry, I didn't realize those were the only `.cpp` files that had `LMI_SO`.